src/components/metadataeditor/metadataeditor.js

```diff
@@ -439,7 +443,6 @@ define(['itemHelper', 'dom', 'layoutManager', 'dialogHelper', 'datetime', 'loadi
 
         var html = metadataInfo.ContentTypeOptions.map(function (i) {
 
```

Comment:
maybe remove those, too?

Reply:
the newlines?

Reply:
yeah, they seem excessive

Reply:
That will be a future change, it requires some fancy changes in the eslint rules to work properly and I don't feel like figuring them out right now.